main/src/ui/contact_details/muc_config_form_provider.vala

```diff
@@ -63,12 +63,12 @@ public class MucConfigFormProvider : Plugins.ContactDetailsProvider, Object {
                     label = _("Occupants may change the subject");
                     break;
                 case "muc#roomconfig_whois":
-                    label = _("Discover real JIDs");
-                    desc = _("Who may discover real JIDs?");
+                    label = _("Permission to view JIDs");
+                    desc = _("Who is allowed to view the occupants' JIDs?");
                     break;
                 case "muc#roomconfig_roomsecret":
                     label = _("Password");
-                    desc = _("Password required for room entry, if any");
```

Comment:
That seems like an incomplete sentence and it was not clear to me what "if any" meant until I saw some of the existing translations